doc/users/config_file.rst

```diff
@@ -73,13 +73,18 @@ Execution
     ``false``; default value: ``true``)
 
 *display_variable*
-    What ``DISPLAY`` variable should all command line interfaces be
-    run with. This is useful if you are using `xnest
-    <http://www.x.org/archive/X11R7.5/doc/man/man1/Xnest.1.html>`_
-    or `Xvfb <http://www.x.org/archive/X11R6.8.1/doc/Xvfb.1.html>`_
-    and you would like to redirect all spawned windows to
-    it. (possible values: any X server address; default value: not
-    set)
+	What ``$DISPLAY`` environment variable should utilize those interfaces
+	that require an X server. These interfaces should have  the attribute
+	``_redirect_x = True``. This option is very useful when the system has
+	an X server listening in the default port 6000, but ``$DISPLAY`` is
+	not defined. In that case, set ``display_variable = :0``. Similarly,
+	it can be used to point X-based interfaces to other servers, like VNC,
+	`xnest <http://www.x.org/archive/X11R7.5/doc/man/man1/Xnest.1.html>`_
+	or `Xvfb <http://www.x.org/archive/X11R6.8.1/doc/Xvfb.1.html>`_
+	and you would like to redirect all spawned windows to
+	it. If not set, nipype will try to configure a new virtual server using
+	Xvfb. (possible values: any X server address; default value: not
+	set)
```

Comment:
This is a little unclear. If I understand it correctly, we use the input `$DISPLAY` variable unless this is set? Under that assumption, here's a proposed rewrite:

```
Override the ``$DISPLAY`` environment variable for interfaces that require an X server.
This option is useful if there is a running X server, but ``$DISPLAY`` was not defined
in nipype's environment. For example, if an X server is listening on the default port of
6000, set ``display_variable = :0`` to enable nipype interfaces to use it. It may also
point to displays provided by VNC, `xnest
<http://www.x.org/archive/X11R7.5/doc/man/man1/Xnest.1.html>`_ or `Xvfb
<http://www.x.org/archive/X11R6.8.1/doc/Xvfb.1.html>`_. If neither ``display_variable``
nor the ``$DISPLAY`` environment variable are set, nipype will try to configure a new
virtual server using Xvfb. (possible values: ...)
```

Reply:
You are right. `display_variable` should override `$DISPLAY`. I'll change the code and use your proposed text for the documentation.